src/content/blog/astro-4130.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Astro 4.13"
+description: Astro 4.13 is now available! This release includes stabilized experimental features, logging improvements, and more.
+publishDate: "August 1, 2024"
+authors:
+  - erika
+coverImage: "/src/content/blog/_images/astro-4130/header_astro_4_13.webp"
+socialImage: "/src/content/blog/_images/astro-4130/og_image_4_13.webp"
+lang: "en"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+
+<BlogSummary>
+  Astro 4.13 is out now! This release includes stabilized experimental features, logging improvements, and more.
+</BlogSummary>
+
+This release includes the following highlights:
+
+- [**Stable: Request Rewriting**](#stable-request-rewriting)
+- [**Stable: Content Collections JSON Schemas**](#stable-content-collections-json-schemas)
+- [**Logging Improvements**](#logging-improvements)
+- [**Support for remote LibSQL databases**](#support-for-remote-libsql-databases)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Stable: Request Rewriting
+
+[In Astro 4.8](https://astro.build/blog/astro-480/#experimental-request-rewriting), we added experimental support for request rewriting. In Astro 4.13, we're excited to announce that request rewriting is now stable! This feature allows you to rewrite incoming requests to different paths or URLs, enabling you to create custom routing rules for your site.
+
+If you were using request rewriting in Astro 4.8, you can now remove the `experimental.rewrites` option from your Astro config. The feature is now enabled by default.
+
+```diff
+import { defineConfig } from 'astro';
+
+export default defineConfig({
+-  experimental: {
+-    rewriting: true
+-  }
+})
+```
+
+As a reminder, request rewriting is a powerful feature that can be used to render another page without changing the URL of the browser in Astro pages and endpoints. For more information on this feature, check out the [Astro documentation](https://docs.astro.build/en/guides/routing/#rewrite).
```

Comment:
```suggestion
Request rewriting is a powerful feature that can be used to render another page without changing the URL of the browser in Astro pages and endpoints. For more information on this feature, check out the [Astro routing documentation on rewrites](https://docs.astro.build/en/guides/routing/#rewrite).
```
When I saw "**As a reminder**, rewriting is a powerful feature..." it kind of sounded like a warning and I was kind of expecting, "only use this if you know what you're doing."  :smile: 
 
I think you could do without, or if you want to stir up excitement with a conclusion kind of sentence,  something like:

"We are excited to see this powerful feature leave its experimental phase..."
"We are excited to make this powerful feature available to everyone..."

(Also, probably helpful to distinguish to avoid having multiple links saying "Astro Documentation".)

If you wanted a sentence about a specific use case, or a simple visual code sample showing usage, not just the config... something like one of these from the docs PR could do:

> Rewrites can be useful for showing the same content at multiple paths (e.g. `/products/shoes/men/` and `/products/men/shoes`/) without needing to maintain two different source files.

and/or

>Rewrites are also helpful to render alternative localized content, such as displaying the `/es/` version of a page when the `/es-CU/` (Cuban Spanish) URL path is visited:

```astro
---
// src/pages/es-cu/articles/introduction.astro
return Astro.rewrite("/es/articles/introduction")
---
```

Reply:
It was more so about saying "If you were not already aware, rewriting is a.."